compiler/tests/evm_contracts.rs

```diff
@@ -224,12 +227,12 @@ fn bytes_token(s: &str) -> ethabi::Token {
     ethabi::Token::FixedBytes(ethabi::FixedBytes::from(s))
 }
 
-fn u256_array_token(v: Vec<usize>) -> ethabi::Token {
-    ethabi::Token::FixedArray(v.into_iter().map(|n| uint_token(n)).collect())
+fn u256_array_token(v: &[usize]) -> ethabi::Token {
```

Comment:
But we still have some leftover single letter variable names today...we should adjust all of them (but doesn't have to be in this PR)